packages/create-plugin/src/commands/generate/prettify-files.ts

```diff
@@ -4,13 +4,13 @@ import fs from 'node:fs';
 
 const exec = promisify(nodeExec);
 
-export async function prettifyFiles(exportPath: string) {
+export async function prettifyFiles(exportPath: string, packageManagerExecute = 'npx') {
   if (!fs.existsSync(exportPath)) {
     return '';
   }
 
   try {
-    let command = 'npx -y prettier@2 . --write';
+    let command = `${packageManagerExecute} -y prettier@2 . --write`;
```

Comment:
Do we need to specify the package manager to use here? `npx` will always be available regardless of the packageManager the project uses and with `-y` will silently grab the version from npmjs if it's not available as a local binary already. If we want to improve this, maybe it's worth checking the package.json to understand which version should be used instead of assuming it's `2`. Maybe projectUsesPrettier could be used here to return the version else it bails if not found?

Reply:
>On the other hand, the same function prettifyFiles is used during create-plugin generate. In this phase, we don't have a package.json to parse and check the prettier version, so version 2 should be used.

I don't think this is correct. We should have a package.json on the filesystem before we call `prettifyFiles`. When generate is run it awaits `generateFiles` which awaits `writeFile` which writes all the files to the disk. Only after that happens do we call `prettifyFiles`.